python/tvm/topi/generic/conv2d.py

```diff
@@ -180,7 +180,7 @@ def schedule_conv_NCHWc_cpu_common_int8(
     ow_chunk, ow_block = s[CC].split(ow, factor=reg_n)
 
     assert oc_bn % int32_lanes == 0
-    assert ic_bn % 4 == 0  # 4 (u)int8 elements in (u)int32
+    assert ic_bn % int8_elems == 0  # (u)int8 elements in (u)int32
```

Comment:
Are you sure that this comment is still valid? I think it might be removed, but otherwise I would write: "input channels should be a multiple of the numer of int8 elements we read"

Reply:
@giuseros ,

* In short it mean: "int8 element, booth uint8(weight), uint8(data) products are accumulated up into int32 lanes."
* Comment still valid,```ic_bn``` should be times of ```int8_elems``` (inner loop cannot be tensorized otherwise).